### PR TITLE
A0-3266: Add featurenet workflows

### PR DESCRIPTION
**Pull request by MikolajGasior**

### CODE REVIEW

**Comment by MikolajGasior** [comment]
Name fixed.

**Comment by MikolajGasior** [comment]
If `featurenet-create-from-branch` is about to be dispatchable then we have to change it a bit and it needs another additional parameter on which node image should be taken, and actually that should be the name because FE images are stored in a separate registry.

**Comment by MikolajGasior** [comment]
I don't see a reason why wouldn't we leave these workflows dispatchable in this repository.  This way we can easily test them.  However, I'm happy to change that.  We can also disable these workflows in the repository.

File: .github/workflows/featurenet-create-from-net.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+name: FE - Create from net
+
+on:
+  workflow_dispatch:
```
Comment (Marcin-Radecki) [inline .github/workflows/featurenet-create-from-net.yml:5]: I think it is enough to have only `workflow_call` in ths repo - it's not intuitive for a user to think what will happen if I trigger workflow from actions; same comment for other workflows in this PR

File: .github/workflows/_featurenet-create-from-branch.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          if [[
+          ! "${{ inputs.featurenet-name }}" =~ ^[a-z0-9][a-z0-9\-]{4,48}$
+          ]]
+          then
+            echo "!!! Invalid featurenet name"
+            exit 1
+          fi
+        # yamllint enable rule:line-length
+
+  get-final-names:
```
Comment (Marcin-Radecki) [inline .github/workflows/_featurenet-create-from-branch.yml:45]: I guess we do not need that job now, as we do not prefix `updatenet` anywhere. So plain `${{ inputs.featurenet-name }}` would suffice; this comment applies to other files as well

**Review by Marcin-Radecki — APPROVED** [review]

File: .github/workflows/featurenet-create-from-net.yml (hunk 1 above)
Comment (MikolajGasior) [reply .github/workflows/featurenet-create-from-net.yml]: You're right, I have removed `workflow_dispatch` from all the workflow.  Let's just have them includable.

**Review by Marcin-Radecki — APPROVED** [review]

**Comment by MikolajGasior** [comment]
Validator is failing and this is all right, there is no `v3` tag yet.

**Comment by MikolajGasior** [comment]
I'll run the test for the last time ([run here](https://github.com/Cardinal-Cryptography/ops-scripts/actions/runs/6402703385)) then merge it and tag with v3.

**Pull request merged**